config/cloud_controller.yml

```diff
@@ -393,3 +393,4 @@ max_manifest_service_binding_poll_duration_in_seconds: 60
 update_metric_tags_on_rename: true
```

Comment:
**thought** In terms of docs, is there anything we think we need to add so that its not unexpected that `STOPPING` state won't be available if this flag is off. Is there any precedence here?

Reply:
I don't think we usually document these types of configurations elsewhere.  We also don't have much documentation about what to expect from the process stats endpoint in general: https://v3-apidocs.cloudfoundry.org/version/3.176.0/index.html#get-a-process

Reply:
I suppose the docs do say `The state of the instance; valid values are RUNNING, CRASHED, STARTING, STOPPING, DOWN` which is not a lie, just in some TAS deployments stopping won't appear.